File: Greedy/Greedy_숫자카드게임2.cpp
#include <iostream>
using namespace std; 

int n,m;
int result;

int main() {
  //n,m을 공백을 기준으로 입력받기 
  cin >> n >> m;

  for(int i=0; i<n; i++){
    int min_value = 100;
    for(int j=0; j<m; j++){
      int x;
      cin >> x;
      min_value=min(min_value, x);
    }
    result = max(result, min_value);
  }
  cout << result <<endl;
}

File: Greedy/Greedy_숫자카드게임2.py
## 이중 반복문 구조 활용
n, m = map(int, input().split())

result = 0 

for i in range(n):
  data=list(map(int, input().split()))
  min_value = 100
  for a in data:
    min_value = min(min_value,a)
  
  result = max(result, min_value)

print(result)
